src/main/java/org/broadinstitute/hellbender/tools/spark/validation/CompareDuplicatesSpark.java

```diff
@@ -30,26 +30,72 @@
 import java.util.Set;
 
 /**
- * The goal of this program is to load two potentially identical BAMs and determine if the BAMs contain the same
- * reads marked as duplicates. It fails if the BAMs don't have the same number of reads or total duplicates.
+ * Determine if two potentially identical BAMs have the same duplicate reads. This tool is useful for checking if two
+ * BAMs that seem identical have the same reads marked as duplicates.
+ *
+ * <p>It fails if at least one of the following conditions is true of the two BAM files:</p>
+ *
+ * <ul>
+ *     <li>Different number of primary mapped reads</li>
+ *     <li>Different number of duplicate reads (as indicated by the SAM record flag)</li>
+ *     <li>Different reads mapped to some position in the reference</li>
+ * </ul>
+ *
+ * <p>The tool gathers the mapped reads together into groups that belong to the same library and map to the same
+ * position and strand in the reference. If the tool does not fail, then it reports the number of these groups with
+ * the following properties:</p>
+ *
+ * <ul>
+ *     <li>SIZE_UNEQUAL: different number of reads</li>
+ *     <li>EQUAL: same reads and same duplicates</li>
+ *     <li>READ_MISMATCH: reads with different names</li>
+ *     <li>DIFFERENT_REPRESENTATIVE_READ: same reads and number of duplicates, but one or more duplicate reads are different</li>
+ *     <li>DIFF_NUM_DUPES: same reads but different number of duplicates</li>
+ * </ul>
+ *
+ * <h3>Input</h3>
+ *
+ * <ul>
+ *     <li>Two BAM files</li>
+ * </ul>
+ *
+ * <h3>Output</h3>
+ *
+ * <p>Results are printed to the screen</p>
+ *
+ * <h3>Usage example</h3>
+ * <pre>
+ * gatk CompareDuplicatesSpark \
+ *     -I input_reads_1.bam \
+ *     -I2 input_reads_2.bam
+ * </pre>
```

Comment:
Given I can run this command without specifying any spark args, add the blurb:

```
This tool can be run without explicitly specifying Spark options. That is to say, the given example command without Spark options will run locally. See <a href ="https://software.broadinstitute.org/gatk/documentation/article?id=10060">Tutorial#10060</a> for an example of how to set up and run a Spark tool on a cloud Spark cluster.
```
 
If possible, a Spark-enabled command would be nice to add. But the above blurb should be sufficient for now.

Reply:
Added the blurb.